src/history/hs.c

```diff
@@ -222,10 +222,6 @@ __wt_hs_cursor_open(WT_SESSION_IMPL *session)
       session, ret = __wt_open_cursor(session, WT_HS_URI, NULL, open_cursor_cfg, &cursor));
     WT_RET(ret);
 
-    /*
-     * Set the flag to stop creating snapshots for history store cursors
-     */
-    F_SET((WT_CURSOR_BTREE *)cursor, WT_CBT_NO_TXN);
```

Comment:
@tetsuo-cpp FYI - Removing this setting of flag might increase the number of `txn global read lock acquisitions` and `txn global write lock acquisitions locks` in the stats.

Reply:
Is that still the case even with #5868 merged?
My understanding is that this is caused by acquiring snapshots. But since we're running at read-uncommitted now, we're just going to be pinning ids without going through the `__wt_txn_get_snapshot` code path.

Reply:
> __wt_txn_get_snapshot

@tetsuo-cpp : Sorry, I didn't check the [#5868](https://github.com/wiredtiger/wiredtiger/pull/5868/files) merge changes. Changes LGTM.